q2_diversity/plugin_setup.py

```diff
@@ -205,7 +205,11 @@
 plugin.methods.register_function(
     function=q2_diversity.pcoa,
     inputs={'distance_matrix': DistanceMatrix},
-    parameters={},
+    parameters={
+        'method': Str,
+        'number_of_dimensions': Int % Range(0, None),
+        'inplace': Bool
+    },
```

Comment:
Add corresponding `parameter_descriptions` for these attributes. You'll definitely want to explain the behavior of 0 for `number_of_dimensions`